patches/net/minecraft/world/entity/player/Player.java.patch

```diff
@@ -160,20 +170,45 @@
                  if (this.useItem.isEmpty()) {
                      if (interactionhand == InteractionHand.MAIN_HAND) {
                          this.setItemSlot(EquipmentSlot.MAINHAND, ItemStack.EMPTY);
-@@ -944,10 +_,13 @@
+@@ -943,16 +_,18 @@
+ 
      @Override
      protected void actuallyHurt(DamageSource p_36312_, float p_36313_) {
-         if (!this.isInvulnerableTo(p_36312_)) {
-+            p_36313_ = net.neoforged.neoforge.common.CommonHooks.onLivingHurt(this, p_36312_, p_36313_);
-+            if (p_36313_ <= 0) return;
-             p_36313_ = this.getDamageAfterArmorAbsorb(p_36312_, p_36313_);
-             p_36313_ = this.getDamageAfterMagicAbsorb(p_36312_, p_36313_);
-             float f1 = Math.max(p_36313_ - this.getAbsorptionAmount(), 0.0F);
-             this.setAbsorptionAmount(this.getAbsorptionAmount() - (p_36313_ - f1));
-+            f1 = net.neoforged.neoforge.common.CommonHooks.onLivingDamage(this, p_36312_, f1);
-             float f = p_36313_ - f1;
-             if (f > 0.0F && f < 3.4028235E37F) {
-                 this.awardStat(Stats.DAMAGE_ABSORBED, Math.round(f * 10.0F));
+-        if (!this.isInvulnerableTo(p_36312_)) {
```

Comment:
Same comment about the removal of `isInvulnerableTo` in the earlier `actuallyHurt` patch